Modules/_interpretersmodule.c

```diff
@@ -885,34 +904,23 @@ interp_create(PyObject *self, PyObject *args, PyObject *kwds)
 }
 
 
-PyDoc_STRVAR(create_doc,
-"create([config], *, reqrefs=False) -> ID\n\
-\n\
-Create a new interpreter and return a unique generated ID.\n\
-\n\
-The caller is responsible for destroying the interpreter before exiting,\n\
-typically by using _interpreters.destroy().  This can be managed \n\
-automatically by passing \"reqrefs=True\" and then using _incref() and\n\
-_decref() appropriately.\n\
-\n\
-\"config\" must be a valid interpreter config or the name of a\n\
-predefined config (\"isolated\" or \"legacy\").  The default\n\
-is \"isolated\".");
+/*[clinic input]
+_interpreters.destroy
+    id: object
+    /
```

Comment:
`id` is not positional-only.

Reply:
We usually ensure that that conversion to Argument Clinic does not change the behavior.

We can make the change more gradual, using `/ [from ...]`. Please do this in a separate PR.